### PR TITLE
HomeWork3

### PR DESCRIPTION
**Pull request by ShytN1k**

### CODE REVIEW

File: src/Animals/Ostrich.php [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Animals;
+
+class Ostrich extends Bird
+{
+    private $speed;
```
Comment (dchabanenko) [inline src/Animals/Ostrich.php:7]: I have seen the same code in SpeedTrait Why do you use copy-paste? Use trait one more time!

Comment (ShytN1k) [inline src/Animals/Ostrich.php:7]: But I have changed the logic of the setSpeed method in the Ostrich class . So I can't use SpeedTrait. Am I wrong?

Comment (dchabanenko) [inline src/Animals/Ostrich.php:7]: Ok, I got it. It's up to you, you can add logics according to your needs.

Comment (dchabanenko) [inline src/Animals/Ostrich.php:7]: by the way, you have a chance to check the priority of traits and usual methods. Try to use trait, and add your own setSpeed(...) method. What method will be used in this case? (in separate branch of course)

Comment (ShytN1k) [inline src/Animals/Ostrich.php:7]: Thanks. I will check this soon.

File: composer.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "autoload": {
+    "psr-4": {
+      "Animals\\": "src/Animals"
+    }
+  }
+}
```
Comment (spolischook) [inline composer.json:7]: add empty line on the end of file

Comment (ShytN1k) [inline composer.json:7]: I've made this file manually. Must I make an empty line on the end of every file? Is it some standard or something else? I haven't seen any notes about this before. 
I will do those empty lines in future.

Comment (spolischook) [inline composer.json:7]: Just keep in mind about that. It's not a standard but good practice

Comment (spolischook) [inline composer.json:7]: in all files

**Pull request merged**